### DISCUSSION TITLE
How can md2po keep the original file's string order intact in the gettext file when updating?

### DISCUSSION THREAD

**Discussion by SantosSi** [mondeja/mdpo#258] [Q&A]
When a string in the original md file is changed, the new string and the old string (commented with #~) are appended to the end of the po/pot file. This is difficult when translating a document because of the loss of the context. To avoid manual sorting, is there a way to keep the original order?

**Comment by mondeja** [comment]
Nope, mainly because the messages of the original PO file could be unordered before updating. The context of a translation must not be managed by their order but using [`mdpo-translator` comments](https://mondeja.github.io/mdpo/latest/commands.html#including-comments-for-translators).
